### PR TITLE
Add alerts for build-cop

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

File: cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            expr: |||
+              sum(rate(prowjob_state_transitions{job="plank",job_name=~"%s",job_name!~"rehearse.*",state="success"}[30m]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"%s",job_name!~"rehearse.*",state=~"success|failure"}[30m])) < 1
+            ||| % [job_name_regex, job_name_regex],
+            'for': '5m',
+            labels: {
+              severity: 'slack',
+              team: 'build-cop',
+            },
+            annotations: {
+              message: '%s jobs have been below the target (100%%) for 5 minutes.' % job_name_regex,
```
Comment (stevekuznetsov) [inline cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: Can we get the regex in a code block?

```
'`%s` jobs have been below the target
```

File: cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            },
+          }
+          for job_name_regex in ['branch-.*-images', 'release-.*-4.1', 'release-.*-4.2', 'release-.*-upgrade.*']
```
Comment (stevekuznetsov) [inline cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: Are these all actually 100%? Do we not have e2e-aws ones anymore?

Comment (hongkailiu) [reply cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: No ... nothing about e2e-aws any more. @bparees updated the gdoc and build-cop board already.

File: cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet (hunk 1 above)
Comment (hongkailiu) [reply cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: Trying.

File: cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet (hunk 2 above)
Comment (bparees) [reply cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: the release jobs include e2e-aws execution.

Comment (bparees) [reply cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: e.g. `release-openshift-ocp-installer-e2e-aws-4.1` which is picked up by `release-.*-4.1`

File: cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet (hunk 1 above)
Comment (hongkailiu) [reply cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet]: https://coreos.slack.com/archives/CHY2E1BL4/p1561141209042700
Done

**Pull request merged**